src/log/ModDaemon.cc

```diff
@@ -298,29 +298,31 @@ logfile_mod_daemon_writeline(Logfile * lf, const char *buf, size_t len)
         }
         return;
     }
-    /* Append this data to the end buffer; create a new one if needed */
+
     /* Are we eol? If so, prefix with our logfile command byte */
+    if (ll->eol == 1) {
+	logfile_mod_daemon_append(lf, "L", 1);
+	ll->eol = 0;
+    }
+
+    /* Append this data to the end buffer; create a new one if needed */
     logfile_mod_daemon_append(lf, buf, len);
 }
 
 static void
 logfile_mod_daemon_linestart(Logfile * lf)
 {
-    l_daemon_t *ll = static_cast<l_daemon_t *>(lf->data);
```

Comment:
This line also needs restoring to fix the build errors

Reply:
Of course, I'm sorry I didn't notice that myself.  Done.